drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -759,6 +751,16 @@ def _add_msg_if_exists(ids_list, message_template, message_class):
                 grade_feedback_class)
             )
 
+        problem_feedback_class = None
+
+        if self.attempts_remain and (misplaced_ids or missing_ids):
+            problem_feedback_message = self.data['feedback']['start']
+        else:
+            problem_feedback_message = self.data['feedback']['finish']
+            problem_feedback_class = 'finish'
```

Comment:
To match the current format, let's add something like `FeedbackMessages.MessageClasses.FINAL_FEEDBACK = "final"` and use `feedback_msgs.append(FeedbackMessage(problem_feedback_message, problem_feedback_class))` to retrieve this class.

Reply:
done